react/features/chat/components/native/ChatButton.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import type { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+
+import {
+    AbstractButton,
+    type AbstractButtonProps
+} from '../../../base/toolbox';
+
+import { toggleChat } from '../../actions';
+import { getUnreadCount } from '../../functions';
+
+type Props = AbstractButtonProps & {
+
+    /**
+     * Display Chat.
+     *
+     * @protected
+     */
+    _onDisplayChat: Function,
+
+    /**
+     * The unread message count.
+     */
+    _unreadMessageCount: number
+};
+
+/**
+ * Implements an {@link AbstractButton} to open the chat screen on mobile.
+ */
+class ChatButton extends AbstractButton<Props, *> {
+    accessibilityLabel = 'toolbar.accessibilityLabel.chat';
+    iconName = 'chat';
+    label = '';
+    toggledIconName = 'chat1';
```

Comment:
can we call this `chat-unread` perhaps?

Reply:
Sure, actually a good idea. I was just not brave enough to edit the generated selection.json. But #YOLO